project/metals.sbt

```diff
@@ -0,0 +1,4 @@
+// DO NOT EDIT! This file is auto-generated.
```

Comment:
Should this file be in .gitignore?

Reply:
Anyone that uses VS code and metals will get this generated, it doesn't affect the build it's just for the ide. But i'll remove it and add it to the .gitignore anyway